app/main.py

```diff
@@ -1,3 +1,71 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+from datetime import datetime
+
+from app.customer import Customer
+from app.car import Car
+from app.shop import Shop
+
+
+def shop_trip() -> None:
+    with open("app/config.json", "r") as f:
+        data = json.load(f)
+
+    fuel_price = data["FUEL_PRICE"]
+    customers = []
+    shops = []
+
+    for customer in data["customers"]:
+        customer_instance = Customer(
+            customer["name"],
+            customer["product_cart"],
```

Comment:
The key `product_cart` is used here, but in the `Customer` class, it is defined as `products_cart`. Ensure consistency in naming to avoid runtime errors.